dev/script/generate-config/service-config.js

```diff
@@ -133,6 +144,7 @@ function getMediaApiConfig(config) {
         |quota.store.key="rcs-quota.json"
         |security.cors.allowedOrigins="${getCorsAllowedOriginString(config)}"
         |metrics.request.enabled=false
+        |dynamo.table.softDelete.metadata="SoftDeletedMetadataTable"
```

Comment:
this config property was moved to common in https://github.com/guardian/grid/commit/18896c4b98c292a16d82b8ababa938f77a703a9d as its also needed by the reaper functionality in `thrall`

Reply:
Thanks Tom - will take it out - just trying to rebase now to make sure everything is up to date before I move it out of draft